helm/cluster-resources/templates/coredns-np.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  egress:
+  - {}
+  ingress:
+  - ports:
+    - port: 1053
+      protocol: UDP
+    - port: 1053
+      protocol: TCP
+    - port: 53
+      protocol: UDP
+    - port: 53
+      protocol: TCP
+  - ports:
+    - port: 9153
+      protocol: TCP
+  podSelector:
+    matchLabels:
+      k8s-app: kube-dns
```

Comment:
We might change this in the future. Could be nice to make it configurable.

Reply:
I don't consider this solution long-term so I don't really want to add all the bells and whistles.